scripts/gitlab/build_and_test.sh

```diff
@@ -142,6 +134,19 @@ then
     tree Testing
     cp Testing/*/Test.xml ${project_dir}
 
+    # Convert CTest xml to JUnit (on toss3 only)
+    if [[ ${sys_type} == *toss_3* ]]; then
+        if [[ -n ${py_env_path} ]]; then
+            . ${py_env_path}/bin/activate
+
+            python3 ${project_dir}/scripts/gitlab/convert_to_junit.py \
+            ${project_dir}/Test.xml \
+            ${project_dir}/scripts/gitlab/junit.xslt > ${project_dir}/junit.xml
+        else
+            echo "ERROR: needs python env with lxml, please set PYTHON_ENVIRONMENT_PATH"
+        fi
+    fi
+
```

Comment:
This actually results in the error message.
However, don't spend time fixing it. I will, using the new method introduced by Chris White in BLT.